liqd_product/apps/cms/pages/templates/liqd_product_cms_pages/home_page.html

```diff
@@ -17,13 +17,6 @@ <h1 class="homepage-title homepage-title--small">Beteiligung.in/<br><span class=
 
             <div>{{ page.body|richtext }}</div>
 
-            <form method="POST" class="homepage-form">
```

Comment:
i'd remove the accompanying css from _cms.scss, too

Reply:
done!